creator-node/scripts/dev-server.sh

```diff
@@ -2,7 +2,7 @@
 set -o xtrace
 set -e
 
-link_libs=false
+link_libs=true
```

Comment:
@dmanjunath why'd you set this to false? it was p useful having it defaulted to true

Reply:
Node 14 upgrade. the libs being built with a different node version threw an error

Reply:
ah right, will revert for now then